staker/block_validator.go

```diff
@@ -1208,3 +1208,9 @@ func (v *BlockValidator) WaitForPos(t *testing.T, ctx context.Context, pos arbut
 		}
 	}
 }
+
+func (v *BlockValidator) GetValidated() arbutil.MessageIndex {
+	v.reorgMutex.RLock()
```

Comment:
do we need the reorg lock here?
`v.validated()` does an atomic read, so I think we should be safe without it

Reply:
Based on https://github.com/OffchainLabs/nitro/blob/master/staker/block_validator.go#L63 it seems like atomic.Load as well as reorg-read is needed to access it.

Reply:
ah.. I you're right. Looks like it is needed to pause other things when a reorg is happening (?) seems ok.